setup.py

```diff
@@ -38,7 +38,7 @@
         'jedi>=0.14.1,<0.16',
         'python-jsonrpc-server>=0.3.0',
         'pluggy',
-        'ujson<=1.35'
+        'ujson<=1.35; platform_system=="Linux"'
```

Comment:
I am not an expert here, but will this prevent mac users form using ujson as well? 
Something like `platform_system != "Windows"`(pseudo, as I am not sure how to properly select it) maybe?

Reply:
same for `jsonrpc` package (where it is actually more important than here)

Reply:
Yep you're right @mpanarin it should be changed for either `platform_system != "Windows"` or something like `platform_system == "Linux" or ´platform_system == "Darwin"` (here and over palantir/python-jsonrpc-server#26)

Reply:
@dalthviz, I agree with @mpanarin. Please change this condition to `platform_system!="Windows"`